app/views/candidate_mailer/chase_reference.text.erb

```diff
@@ -1,9 +1,27 @@
-<%= @reference.name %> has not responded yet.
+<% if @application_form.show_new_reference_flow? %>
+  Dear <%= @application_form.first_name %>
 
-You can add as many referees as you like to increase the chances of getting 2 references quickly.
+  You asked <%= @reference.name %> for a reference for your teacher training application. They have not replied yet.
```

Comment:
I think we might need to remove all the leading spaces, as Notify treats all the text as Markdown, and 2+ leading spaces does something odd?

```suggestion
You asked <%= @reference.name %> for a reference for your teacher training application. They have not replied yet.
```

Reply:
yeah that's right! Have adjusted after seeing it was all a bit of in the mailer preview 🥴